src/Utils/AuthorizationUtils.ts

```diff
@@ -40,3 +41,21 @@ export function decryptJWTToken(token: string) {
 
   return JSON.parse(tokenPayload);
 }
+
+export function getMsalInstance() {
+  const config: msal.Configuration = {
+    cache: {
+      cacheLocation: "localStorage",
+    },
+    auth: {
+      authority: "https://login.microsoftonline.com/common",
+      clientId: "203f1145-856a-4232-83d4-a43568fba23d",
+    },
+  };
+
+  if (process.env.NODE_ENV === "development") {
+    config.auth.redirectUri = "https://dataexplorer-dev.azurewebsites.net";
+  }
+  const msalInstance = new msal.PublicClientApplication(config);
```

Comment:
This will new up an instance every time its called. Is that needed? Why not create it at the top level and export as a const?

Reply:
I did this to have one instance in Explorer and one in HostedExplorer - not sure if it's needed, a const might work fine since the config is the same